docs/reference/builder.md

```diff
@@ -1291,6 +1291,74 @@ The `STOPSIGNAL` instruction sets the system call signal that will be sent to th
 This signal can be a valid unsigned number that matches a position in the kernel's syscall table, for instance 9,
 or a signal name in the format SIGNAME, for instance SIGKILL.
 
+## HEALTHCHECK
+
+The `HEALTHCHECK` instruction has two forms:
+
+* `HEALTHCHECK [OPTIONS] CMD command` (check container health by running a command inside the container)
+* `HEALTHCHECK NONE` (disable any healthcheck inherited from the base image)
```

Comment:
I'm inclined to agree with this. It seems powerful, but also annoying.
And nulling out a healthcheck should really be addressed along with other declared items as well (ENV, VOLUME, EXPOSE, etc.)